uber/templates/preregistration/hotel.html

```diff
@@ -0,0 +1,49 @@
+{% extends "./preregistration/preregbase.html" %}
+{% block title %}Request Hotel Booking Info{% endblock %}
+{% block backlink %}{% endblock %}
+{% block content %}
+
+<div class="masthead"></div>
+<div class="panel panel-default">
+  <div class="panel-body">
+    {{ macros.prereg_wizard(c.PAGE_PATH, c.PREREG_REQUEST_HOTEL_INFO_ENABLED) }}
+
+    <h3>Would you like to be informed when hotel booking is available?</h3>
+    <p>
+      In an effort to help hotel availability, we will initially only release
+      hotel booking information to people registered for MAGFest. Checking this
+      box will put you on the list of people to receive this information before
+      it's posted publicly.
+    </p>
+    {% if is_group_leader %}
+      <p>
+        As a group leader, make sure your group members claim their badges so
```

Comment:
This wording may lead people to assume that simply claiming a badge will earn you a link. Might want to update it to say something like "make sure your group members claim their badges so they can also opt in to receive the email."